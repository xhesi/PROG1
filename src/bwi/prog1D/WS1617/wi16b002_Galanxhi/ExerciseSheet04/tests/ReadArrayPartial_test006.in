33
44
33
55
6
8
6
2
3
2
