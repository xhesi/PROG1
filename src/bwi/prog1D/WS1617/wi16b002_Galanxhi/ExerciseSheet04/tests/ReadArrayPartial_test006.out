1. number: 2. number: 3. number: 4. number: 5. number: 6. number: 7. number: 8. number: 9. number: 10. number: 10 number(s) entered.
[0]: 33
[1]: 44
[2]: 33
[3]: 55
[4]: 6
[5]: 8
[6]: 6
[7]: 2
[8]: 3
[9]: 2
Output finished.